### PR TITLE
Re enabling persistent volume claims

### PR DESCRIPTION
**Pull request by Schife**
Instead of making use of azure disks, I've created a new type of storage class centered around azure files which are a less expensive form of storage.

### CODE REVIEW

File: buildSrc/src/main/groovy/net/corda/testing/KubesTest.java [hunk 1]
```diff
@@ -116,7 +116,13 @@ private static String rnd64Base36(Random rnd) {
             boolean printOutput,
             int numberOfRetries
     ) {
-        return executorService.submit(() -> buildRunPodWithRetriesOrThrow(client, namespace, numberOfPods, podIdx, podName, printOutput, numberOfRetries));
+        return executorService.submit(new Callable<KubePodResult>() {
```
Comment (zkiss) [inline buildSrc/src/main/groovy/net/corda/testing/KubesTest.java]: seems like this could be a lambda

**Review by zkiss — DISMISSED** [review]

File: buildSrc/src/main/groovy/net/corda/testing/KubesTest.java (hunk 1 above)
Comment (Schife) [reply buildSrc/src/main/groovy/net/corda/testing/KubesTest.java]: I'm just reintroducing the code that was already there

**Review by roastario — CHANGES_REQUESTED** [review]
Looks good Raz, could you just commit the .yaml that creates the storage class?

File: buildSrc/src/main/groovy/net/corda/testing/KubesTest.java [hunk 2]
```diff
@@ -306,10 +306,10 @@ private void waitForPodToStart(KubernetesClient client, Pod pod) {
     }
 
     private Collection<File> downloadTestXmlFromPod(KubernetesClient client, String namespace, Pod cp) {
-        String resultsInContainerPath = "/tmp/source/build/test-reports";
+        String resultsInContainerPath = TEST_RUN_DIR + "/test-reports";
         String binaryResultsFile = "results.bin";
         String podName = cp.getMetadata().getName();
-        Path tempDir = new File(new File(getProject().getBuildDir(), "test-results-xml"), podName).toPath();
+        Path tempDir = new File(new File(TEST_RUN_DIR, "test-results-xml"), podName).toPath();
```
Comment (roastario) [inline buildSrc/src/main/groovy/net/corda/testing/KubesTest.java]: this I think is incorrect - we should leave the original path here, as this is the path on the machine we are downloading to.

**Review by roastario — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/groovy/net/corda/testing/KubesTest.java (hunk 2 above)
Comment (Schife) [reply buildSrc/src/main/groovy/net/corda/testing/KubesTest.java]: you are correct

**Review by roastario — DISMISSED** [review]

**Comment by roastario** [comment]
Looks good to me!

**Review by roastario — APPROVED** [review]

**Pull request merged**